public/scripts/search.js

```diff
@@ -79,13 +80,13 @@ export default (history, document, ordering, domUtils) => {
       $orderRelevance.classList.remove('last__button');
       $orderColor.classList.add('last__button');
 
-      // add all icons to the grid again
+      // Add all icons to the grid again
       const $gridChildren = document.querySelector('ul.grid').children;
-      domUtils.replaceChildren(
-        document.querySelector('ul.grid'),
-        getNonIcons($gridChildren).concat($allIcons),
-      );
-      // and reset to the preferred ordering
+      domUtils.replaceChildren(document.querySelector('ul.grid'), [
+        ...getNonIcons($gridChildren),
+        ...$allIcons,
+      ]);
```

Comment:
`concat` is faster and here performance in totally crucial, I've refactored this code several times for this reason. See https://stackoverflow.com/questions/48865710/spread-operator-vs-array-concat

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v51.0.1/docs/rules/prefer-spread.md

But I can ignore the error.

Reply:
Please, ignore it. This would be a huge regression in performance.